### ISSUE TITLE
Text is not Comparable<Text>

### ISSUE THREAD

**Issue by yegor256** [yegor256/cactoos#56]
Let's make sure that `Text extends Comparable<Text>`

**Comment by fabriciofx** [comment]
We should use lombok to implement `compareTo()`?

**Comment by englishman** [comment]
@fabriciofx it'll introduce a heavy external dependency, I guess. In README.md is:

> How to use. **The library has no dependencies.** All you need is this (get the latest version here):

**Comment by fabriciofx** [comment]
@englishman Yeah, you're correct. Thank you for the clarification.

**Comment by fabriciofx** [comment]
@englishman I'm trying solve this issue. At the moment, my `compareTo()` method is:

```java
    public int compareTo(final Text text) {
        try {
            return this.asString().compareTo(text.asString());
        } catch (final IOException ex) {
            throw new IllegalArgumentException(ex);
        }
    }
```
So my question is: if something goes wrong, we should throws an `IllegalArgumentException` or `RuntimeException` encapsulating the `IOException`? WDYT?

**Comment by yegor256** [comment]
@fabriciofx I think we should introduce `UncheckedScalar`, similar to `UncheckedFunc`: #74

**Comment by englishman** [comment]
@fabriciofx don't clearly understand what can be wrong in this `compareTo` code. Do we need here any exception processing?

**Comment by fabriciofx** [comment]
@yegor256 Could you check the latest comments in the PR?

**Comment by fabriciofx** [comment]
Closed #219

**Comment by yegor256** [comment]
@fabriciofx thanks!
